.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: python
 python:
-  - "2.7.6"
   - "2.7"
```

Comment:
Can we keep both 2.7 versions in the list but add then to Travis's [allow_failures](https://docs.travis-ci.com/user/customizing-the-build#jobs-that-are-allowed-to-fail) list? Too many of our users are still on 2.7 to drop visibility entirely.

Reply:
@harman28 Any particular reason why you don't want to drop `2.7.6` (entirely) from list of versions being tested on?

Since [Python `2.7` is still mentioned in .travis.yml](https://github.com/Vibhu-Agarwal/razorpay-python/blob/53291df39b89c3b3c7612bf486f70ba1780bbd55/.travis.yml#L3), the future commits will still be tested on Python `2.7`.
Also, there's been [**no removal** of Python `2.7` versions](https://github.com/Vibhu-Agarwal/razorpay-python/blob/53291df39b89c3b3c7612bf486f70ba1780bbd55/setup.py#L20-L28) from the [setup.py](https://github.com/razorpay/razorpay-python/blob/master/setup.py) in this PR.

PS. It says that [allow_failures](https://docs.travis-ci.com/user/customizing-the-build#jobs-that-are-allowed-to-fail) lets us add _experimental and preparatory_ builds, that is, we expect them to pass at some point if they're able to. But job running on `2.7.6` will **always** fail as it [fails to download Python itself](https://github.com/razorpay/razorpay-python/issues/93).

Reply:
@harman28 Ping!